low-level/files/opt/seagate/sspl/bin/conf_based_sensors_enable

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -e -u -o pipefail
+
+to_lower_case() {
+    tr '[:upper:]' '[:lower:]'
+}
+
+declare -a sed_expr=(-e '/^\[[A-Z]\+\]$/h')
+storage_type="$(provisioner get_setup_info --output json | jq -r .ret.storage_type | to_lower_case)"
+server_type="$(provisioner get_setup_info --output json | jq -r .ret.server_type | to_lower_case)"
+
+declare -A monitor=(
+    [REALSTORSENSORS]="true"
+    [NODEHWSENSOR]="true"
+    [SYSTEMDWATCHDOG]="true" # This is the 'server disks' sensor
+    [RAIDSENSOR]="true"
+    [SASPORTSENSOR]="true"
+    [MEMFAULTSENSOR]="true"
+    [CPUFAULTSENSOR]="true"
+
+)
+
+case "${storage_type}" in
+    virtual) 
+        monitor[REALSTORSENSORS]="false"
+        monitor[SYSTEMDWATCHDOG]="false"
+        ;;
+    jbod)
+        monitor[REALSTORSENSORS]="false"
+        monitor[SYSTEMDWATCHDOG]="false"
+        monitor[RAIDSENSOR]="false"
+        ;;
+esac
+
+case "${server_type}" in
+    virtual)
```

Comment:
@ujjwalpl If server type is virtual, RAID sensor would needed still? meant for server local disk RAID monitoring?

Reply:
RAID may not be there in virtual env necessarily, safe is to keep it disabled by default. These configs can overridden if someone wants to try these sensors still in virtual env, like by editing its value in config file & not running this conf_based_sensors_enable